docs/contributing.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Open blog is before all an open source project to share a publication of blog posts with the community.
+
+> Developed by the community for everyone freely
+
+## Share your blog posts :rocket:
+
+Before adding your blog post to Open blog, you should check those 3 steps.
+
+### Post rules
+
+- Post should follow the [Code of Conduct](https://github.com/open-blog-initiative/open-blog.dev/blob/master/CODE_OF_CONDUCT.md).
+- Post can't be direct advertising.
+- Post can't break any copyright in its content.
+- Post should not be published in another blog.
```

Comment:
???

Reply:
SEO purpose my friend. If you publish exactly the same content in many page. Google won't understand and will say "Duplicate content" 

Here is a solution https://www.yakaferci.com/link-rel-canonical/

Reply:
I don't really agree that SEO should take precedence over freedom 🤔 . I would like to publish my post everywhere I want 😛